tests/test_translation.h

```diff
@@ -179,3 +179,4 @@ TEST_CASE("[Translation] CSV import") {
 } // namespace TestTranslation
 
 #endif // TEST_TRANSLATION_H
+
```

Comment:
There's no reason to change this file in this PR anymore.

Reply:
Removed from PR.